src/ArtifactoryUploader/PackageUploadHelper.cs

```diff
@@ -335,6 +353,10 @@ private static string GetComponentType(Component item)
             {
                 return "CONAN";
             }
+            else if (item.Purl.Contains("debian", StringComparison.OrdinalIgnoreCase))
```

Comment:
You can check with 'pkg:deb/debian' , because sometimes other linux based components can contain Debian names.